platform/android/export/export_plugin.cpp

```diff
@@ -1704,11 +1704,13 @@ void EditorExportPlatformAndroid::get_export_options(List<ExportOption> *r_optio
 	}
 
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug", PROPERTY_HINT_GLOBAL_FILE, "*.keystore,*.jks"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug_keystore_password"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug_user"), ""));
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug_password"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug_user_password"), ""));
```

Comment:
Since property names are changed / added, error messages below should be changed accordingly. e.g.

> "Either Release Keystore, Release User AND Release Password settings must be configured OR none of them."

Reply:
Corresponding processing has been done in the code. When "user_password" is empty, "password" will be used as the default password, which is compatible with the previous processing method of the document author.
Refer to the following code:
```
// Fix export or run program signature issues.
args.push_back("--key-pass");
if (user_password.is_empty()) {
args.push_back("pass:" + password);
} else {
args.push_back("pass:" + user_password);
}
```

Reply:
I mean, this PR replaced "Debug Password" with "Debug Keystore Password" and "Debug User Password", and "Debug User Password" is optional.

The error message here still uses "Debug Password":

> Either Debug Keystore, Debug User AND Debug Password settings must be configured OR none of them.

So I think you should replace the "Debug Password" here with "Debug Keystore Password" too.

Reply:
ok,already edited.

Reply:
Boss, I just found out that you are from China, should I answer in Chinese? English is not much.

Reply:
It's better to use English, so others can understand.